### PR TITLE
[chore] fix loading skeleton in file picker

### PR DESCRIPTION
**Pull request by Kharonus**
- excluded breadcrumbs from being replaced by loading skeleton
- added cancellation to data load, if breadcrumbs action is triggered while loading
- resized skeleton correctly

### CODE REVIEW

File: frontend/src/global_styles/content/work_packages/shared/_file_list.sass [hunk 1]
```diff
@@ -76,6 +76,13 @@
       &-icon
         margin-right: $spot-spacing-0_25
 
+    & &-button
```
Comment (b12f) [inline frontend/src/global_styles/content/work_packages/shared/_file_list.sass:79]: I don't think you need the double & here, it just adds an extra class of specificity

Comment (Kharonus) [reply frontend/src/global_styles/content/work_packages/shared/_file_list.sass:79]: yep, without it, the styles from `.spot-link` are loaded on top of this and thus `display: inline` overwrites the `display: flex` 

and I definitely prefer specific selectors over `!important`

Comment (Kharonus) [reply frontend/src/global_styles/content/work_packages/shared/_file_list.sass:79]: you think it is okay otherwise?

**Review by b12f — APPROVED** [review]
Yes, LGTM

**Pull request merged**